contracts/storage/Storage.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.6.9;
+import "./Structs.sol";
+
+contract Storage is Structs {
+    uint256 public constant MANAGEMENT_KEY = 1;
```

Comment:
useless variable imo, additionally, the value of the variable will not be shared with the proxy storage

Reply:
These variables were already present on the onchainID

Reply:
what do you mean by not be shared with the proxy storage?

Reply:
when you create a proxy, its storage is empty, so the values (1, 2, 3, 4) of the keys here will not be visible for the proxy normally

Reply:
except if you put their initialization on the constructor function

Reply:
this is part of the logic, the keys are visible once the proxy is defined